app/models/match_decisions/six/approve_match_housing_subsidy_admin.rb

```diff
@@ -15,9 +15,9 @@ def label
 
     def label_for_status status
       case status.to_sym
-      when :pending then "#{Translation.translate('Housing Subsidy Administrator')} reviewing match"
-      when :accepted then "Match accepted by #{Translation.translate('Housing Subsidy Administrator')}"
-      when :declined then "Match declined by #{Translation.translate('Housing Subsidy Administrator')}.  Reason: #{decline_reason_name}"
+      when :pending then "#{Translation.translate('Housing Subsidy Administrator Six')} reviewing match"
+      when :accepted then "Match accepted by #{Translation.translate('Housing Subsidy Administrator Six')}"
+      when :declined then "Match declined by #{Translation.translate('Housing Subsidy Administrator Six')}.  Reason: #{decline_reason_name}"
```

Comment:
I think we should add a migration to set the defaults for all of these `Six` specific translations to be translated as the previous text.  So for example `CoC Staff` would be translated as `DND` and `Housing Subsidy Administrator Six` as `Housing Subsidy Administrator`.  That way if anyone _is_ using the route, the names won't change, but we can translate as necessary.

Reply:
@eanders This has been added to the PR